CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+**Our Code of Conduct is available here: https://code-of-conduct.freecodecamp.org/**
```

Comment:
Do we need a separate document for this?  Can we consolidate it into contributing or README?

Reply:
Might as well, it's one of GitHub's officially supported ["community health files"](https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization#supported-file-types)

Reply:
Seems like following the Github pattern is ideal. We can always remove it down the road if a better pattern surfaces.